addons/mrp/views/mrp_production_views.xml

```diff
@@ -28,6 +28,7 @@
                         <button name="button_plan" type="object" string="Plan"/>
                         <button name="do_unreserve" type="object" string="Unreserve"/>
                         <button name="action_cancel" type="object" string="Cancel"/>
+                        <button name="action_assign"  type="object" string="Check availability"/>
```

Comment:
For me, `Cancel` is normally the last button and it's also like that on the spec too.